src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukNotificationBanner.scala.html

```diff
@@ -39,10 +39,12 @@
     titleId.getOrElse("govuk-notification-banner-title")
 }
 
+@hasDisableAutoFocus = @{disableAutoFocus.isDefined}
+
 <div class="@toClasses("govuk-notification-banner", classes, if(bannerType.contains("success"))" govuk-notification-banner--success" else "")"
     role="@{bannerRole}"
     aria-labelledby="@{headingAndTitleId}"
-    data-module="govuk-notification-banner"@if(disableAutoFocus.getOrElse(false)){ data-disable-auto-focus="true"}@toAttributes(attributes)>
+    data-module="govuk-notification-banner"@if(hasDisableAutoFocus){ data-disable-auto-focus="@{disableAutoFocus.get}"}@toAttributes(attributes)>
```

Comment:
same as before, could it be a map instead, because twirl will automatically unwrap options https://www.playframework.com/documentation/2.8.x/ScalaTemplates#Displaying-Scala-types